config/releases.exs

```diff
@@ -205,8 +257,20 @@ config :service_orchestrate, Orchestrate.Application,
     ],
     handlers: [Orchestrate.Event.Handler],
     storage: [
-      module: Brook.Storage.Redis,
-      init_arg: [redix_args: redix_args, namespace: "service:orchestrate:view"]
+      module: Brook.Storage.Postgres,
+      init_arg: [
+        table: "orchestrate_state",
+        postgrex_args: [
+          hostname: "localhost",
+          username: "orchestrate_app_user",
+          password: "orchestrate123",
+          database: "orchestrate_app_state"
+        ],
+        event_limits: %{
+          # "data:something" => 100,
+          # "data:something:else" => 50
+        }
```

Comment:
same here

Reply:
That's what I get for copy/pasting an example.